api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -108,6 +108,29 @@ message Bootstrap {
   // identification purposes (e.g. in generated headers).
   core.v3.Node node = 1;
 
+  // A list of :ref:`Node <envoy_v3_api_msg_config.core.v3.Node>` field names
```

Comment:
It took me a bit to understand that the user can specify any field names in the node message. It wasn't immediately clear to me on first reading. Can you potentially spell this out in more detail? I think it won't be obvious to the user that this is how things are referenced. (I was looking for some field in Node that supplies these values.)